Scheduling — USF-2178 (#715)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/edge-delivery-services/scheduling.mdx

---
title: Scheduled Publishing
description: Schedule content for publishing at a later date and time using AEM Edge Delivery Services.
---

import Link from '@components/Link.astro';
import { Steps, Aside } from '@astrojs/starlight/components';

> Schedule content to publish at a future date and time. Group multiple pages together so they all go live simultaneously.

Reviewer: ```suggestion
Schedule content to publish at a future time. Group multiple pages together so they all go live simultaneously.
```

IMO, date and time is redundant.

## Overview

Scheduled publishing lets you prepare content in advance and control when it goes live:

- **Product launches**: Coordinate content updates across multiple pages to go live at a specific time.
- **Promotions**: Schedule sale pages and banners to publish when your campaign starts.
- **Coordinated releases**: Publish related pages (landing page, product pages, FAQs) together.

## How it works

<Steps>
1. **Register your site**. A one-time setup enables scheduling for your organization and site.
1. **Create content**. Author your pages in Document Authoring (da.live) or your preferred tool.

> 1. **Schedule publishing**. Select the pages you want to publish together and set a future date and time.

Reviewer: ```suggestion
1. **Schedule publishing**. Select the pages you want to publish together and specify when they'll be published.
```

> 1. **Automatic publishing**. Your content goes live at the scheduled time with no manual intervention.

Reviewer: This isn't really a step. Combine with the previous line.
